packages/blade/src/components/Icons/AlertIcon/index.ts

```diff
@@ -1 +0,0 @@
-export { default } from './AlertIcon';
```

Comment:
Wouldn't this be a breaking change? If someone is using this their imports will fail now

Reply:
Should be fine, this isn't merged to `master` yet. I introduced this in PR1 and later realized the icon name is different 😄